bundles/org.openhab.binding.pushbullet/src/main/java/org/openhab/binding/pushbullet/internal/handler/PushbulletHandler.java

```diff
@@ -87,9 +87,35 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void initialize() {
         logger.debug("Start initializing!");
-        config = getConfigAs(PushbulletConfiguration.class);
+        PushbulletConfiguration config = getConfigAs(PushbulletConfiguration.class);
+
+        httpClient.setConfiguration(config);
+
+        if (config.getToken().isEmpty()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Undefined token.");
+            return;
+        }
```

Comment:
Minor, better to first check and then set
```suggestion
        if (config.getToken().isEmpty()) {
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Undefined token.");
            return;
        }
        
        httpClient.setConfiguration(config);
```